packages/webapp/src/components/Table/v2/index.jsx

```diff
@@ -0,0 +1,282 @@
+/*
+ *  Copyright 2023 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+import { useState, useMemo } from 'react';
+import PropTypes from 'prop-types';
+import { useTranslation } from 'react-i18next';
+import Box from '@mui/material/Box';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TablePagination from '@mui/material/TablePagination';
+import TableRow from '@mui/material/TableRow';
+import { BsThreeDots } from 'react-icons/bs';
+import clsx from 'clsx';
+import EnhancedTableHead from './TableHead';
+import Button from '../../Form/Button';
+import styles from './styles.module.scss';
+
+const More = ({ onClickLoadMore, onClickMore, invisibleRowCount, dense, colSpan }) => {
+  const { t } = useTranslation();
+
+  if (invisibleRowCount <= 0) {
+    return null;
+  }
+
+  if (onClickMore) {
+    return (
+      <TableRow>
+        <TableCell colSpan={colSpan} className={clsx(styles.tableCell, dense && styles.dense)}>
+          <button onClick={onClickMore} className={styles.moreButton}>
+            <BsThreeDots />
+            <span>{t('TABLE.NUMBER_MORE', { number: invisibleRowCount })}</span>
+          </button>
+        </TableCell>
+      </TableRow>
+    );
+  }
+
+  return (
+    <TableRow>
+      <TableCell colSpan={colSpan} className={clsx(styles.loadMoreCell)}>
+        <Button
+          className={styles.loadMoreButton}
+          onClick={onClickLoadMore}
+          color={'secondary'}
+          sm={true}
+        >
+          {t('TABLE.LOAD_MORE')}
+        </Button>
+      </TableCell>
+    </TableRow>
+  );
+};
+
+More.propTypes = {
+  onClickLoadMore: PropTypes.func,
+  onClickMore: PropTypes.func,
+  invisibleRowCount: PropTypes.number,
+  dense: PropTypes.bool,
+  colSpan: PropTypes.number,
+};
+
+function descendingComparator(a, b, orderBy) {
```

Comment:
Wondering if something like this should go in a top level utility -- I think I have seen something like this a few times in the code.